app/build.gradle

```diff
@@ -53,7 +53,7 @@ android {
         minSdkVersion 21
         targetSdkVersion 28
         versionCode 113
-        versionName "1.2.8"
+        versionName "1.4.0"
```

Comment:
This change is targeted for 1.4.0 release. Hence the change. (also we are already at 1.3.0 in production I believe)